docs/contributing/roadmap.rst

```diff
@@ -0,0 +1,107 @@
+.. _roadmap:
+
+####################
+Development Roadmap
+####################
+
+**Authors**: Anemoi Technical Subgroup (ATS)
```

Comment:
Thanks for this Daniele! will review it more detail later but this just made me wonder. So far I think we haven't explained the role of ATS within Anemoi. Should we take this opportunity to clarify this as part of the anemoi-docs and explain what are the 'ATS approval needed' labels? I started the draft for that and shared some weeks ago at [ATS](https://docs.google.com/document/d/1a5unI04UtfzMQbZLr7JqqFArRT7sN2PNAe4Nq3P79c4/edit?tab=t.0)

Reply:
I agree, this would be a good opportunity to fill this gap! Should we maybe do it in a separate PR though?

Reply:
maybe you could also bring your draft above in a PR here?

Reply:
This is already added in PR 33